.circleci/cimodel/data/dimensions.py

```diff
@@ -13,3 +13,6 @@
     "3.7",
     "3.8"
 ]
+
+STANDARD_PYTHON_VERSIONS_WINDOWS = STANDARD_PYTHON_VERSIONS.copy()
```

Comment:
I think instead of doing this it'd just be better to remove python 3.5 altogether, wdyt @ezyang ?

cc @malfet

Reply:
I'll remove that.

Reply:
As 3.5 no longer supported, I think we should remove it.